NEWS

```diff
@@ -6,6 +6,14 @@ CHANGES WITH 236 in spe:
 	  has been extended to also set the dummy module option numdummies=0,
 	  resolving issues with the kernel creating dummy0.
 
+        * systemd-resolved now maintains a new dynamic
+          /run/systemd/resolve/stub-resolv.conf compatibility file. It is now
+          recommended to maintain /etc/resolv.conf as a symlink to this new
+          dynamic file. It points at the systemd-resolved stub DNS 127.0.0.53
+          resolver and it includes dynamically acquired search domains. This
+          achieves a more correct DNS resolution by software that bypasses
+          local DNS APIs (e.g. NSS).
```

Comment:
It is unclear what the last "e.g." refers to: is NSS an example of a software that bypasses local APIs or is it an example of those local APIs? (I know the answer, I'm just pointing out that this needs editing.)

Maybe also add a sentence like "See the section /ETC/RESOLVED.CONF in systemd-resolved.service(8) for more information.".

Reply:
I'm not a native english speaker, but I do not think anybody out of the target audience will be confused by this statement as is. And it appears to be grammatically correct to me.